packages/loader/container-loader/src/container.ts

```diff
@@ -1688,7 +1689,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         }
 
         // Forward non system messages to the loaded runtime for processing
-        if (!isSystemMessage(message)) {
+        // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+        if (isRuntimeMessage(message) || isLegacyRuntimeMessage(message)) {
```

Comment:
I'd suggest fire telemetry event when we hit isLegacyRuntimeMessage() && !isRuntimeMessage
That way we can take a look if we ever hit them (in let's say 30-day timeframe). If we hit zero, I'd rather remove it from here.